packaging/leapp.spec

```diff
@@ -203,7 +205,7 @@ install -m 0644 -p man/snactor.1 %{buildroot}%{_mandir}/man1/
 %dir %{_sysconfdir}/leapp/repos.d
 %{_bindir}/leapp
```

Comment:
I'm getting an error on fedora:
```
Checking for unpackaged file(s): /usr/lib/rpm/check-files /home/mbocek/rpmbuild/BUILDROOT/leapp-0.5.0-0.201902191126Z.1c67a50.fedora_builds.fc29.x86_64
error: Installed (but unpackaged) file(s) found:
   /usr/bin/leapp
```

Reply:
Adding the following to the %install section does the trick. It looks abit ugly/hacky though. @pirat89 can chime in.
```
%if 0%{?fedora}
rm -rf %{buildroot}%{_bindir}/leapp
%endif
```

Reply:
yep, the remove like that is valid.